view/frontend/web/js/components/steps/TheShippingStep.vue

```diff
@@ -148,9 +148,15 @@ export default {
           console.error('Error with checking email availability. \n', error)
         })
     },
+    onCountryChange (selectedOption) {
+      this.$store.commit('setLoading', true)
+      this.regions = this.$store.getters.regionsByCountryId(this.address.country_id)
+      this.$store.dispatch('updateShippingMethods', this.address.country_id)
+    },
```

Comment:
Do we need this method. Where do you execute it?

Reply:
It looks like some dead code - deleted

Reply:
👍